README.md

```diff
@@ -107,23 +110,64 @@ return fmt.Errorf("invalid port: %v", err)
 
 ## Apply the patch
 
-To apply the patch, `cd` to your Go project's directory.
-
-```shell
-$ cd ~/go/src/example.com/myproject
-```
-
-Run `gopatch` on the project, supplying the previously written patch with the
-`-p` flag.
-
-```shell
-$ gopatch -p ~/s1028.patch ./...
-```
-
-This will apply the patch on all Go code in your project.
-
-Check if there were any instances of this issue in your code by running
-`git diff`.
+- To apply the patch, `cd` to your Go project's directory.
+
+  ```shell
+  $ cd ~/go/src/example.com/myproject
+  ```
+
+  Run `gopatch` on the project, supplying the previously written patch with the
+  `-p` flag.
+
+  ```shell
+  $ gopatch -p ~/s1028.patch ./...
+  ```
+
+  This will apply the patch on all Go code in your project.
+
+  Check if there were any instances of this issue in your code by running
+  `git diff`.
+- To write the diff, instead, `cd` to your Go project's directory.
```

Comment:
Nit: the sentence looks confusing.
Sounds like:
`To write the diff` => `instead, `cd` to your Go project's directory.`

Maybe delete the part `To write the diff` as it's already mentioned in the section title?

The same applies to `To apply the patch, `cd` to your Go project's directory.`